### ISSUE TITLE
Is there a difference between `external-type` and `type`?

### ISSUE THREAD

**Issue by 13OnTheCode** [azat-io/eslint-plugin-perfectionist#28]
### What rule do you want to change?

sort-imports

### Describe the problem

I apologize for raising a question here because the "Discussions" section is closed.

I noticed that a new type called "external-type" has been added in the source code, which is not mentioned in the documentation. Is there any difference between "external-type" and the existing "type"? If they are the same, would it be possible to update the documentation to promote "external-type" as the primary option and gradually remove "type" in future versions? I find "type" confusing in terms of semantics.

### Code example

```
type Group<T extends string[]> =
  | 'external-type'
  | 'internal-type'
  | 'builtin-type'
  | 'sibling-type'
  | 'parent-type'
  | 'side-effect'
  | 'index-type'
  | 'internal'
  | 'external'
  | T[number]
  | 'sibling'
  | 'unknown'
  | 'builtin'
  | 'parent'
  | 'object'
  | 'index'
  | 'style'
  | 'type'
```

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
`type` is fallback.
Maybe we should remove it in the next version.

**Comment by 13OnTheCode** [comment]
Thank you for @azat-io  answer. Should we update the documentation now to replace "type" with "external-type"?

**Comment by azat-io** [comment]
Yeah, that's a good point

**Comment by azat-io** [comment]
Added in 47b07ccd32400b723c9a03e05a3d67ddcd303916

Released in [v1.5.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v1.5.0)
